### PR TITLE
auth: EOL update after 4.5 release

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Review by omoerbeek — COMMENTED** [review]
Recursor uses the scheme: last release fully supported, last-1 and last-2 get critical updates only. This PR seem a bit more generous.

**Comment by Habbie** [comment]
> Recursor uses the scheme: last release fully supported, last-1 and last-2 get critical updates only. This PR seem a bit more generous.

I 'm switching to that scheme starting with 4.5; apparently I did not do it earlier. Are my calculations correct with that information?

File: docs/appendices/EOL.rst [hunk 1]
```diff
@@ -7,14 +7,13 @@ The release before that gets critical security updates only.
...
 
-The currently supported release train of PowerDNS Authoritative Server is 4.4.
+The currently supported release train of PowerDNS Authoritative Server is 4.5.
 
-PowerDNS Authoritative Server 4.3 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.5 is released.
-It will be end of life after PowerDNS Authoritative Server 4.6 is released.
+PowerDNS Authoritative Server 4.4 will only receive correctness, stability and security updates and will be receiving critical updates only after PowerDNS Authoritative Server 4.6 is released. It will be end of life after PowerDNS Authoritative Server 4.7 is released.
 
-PowerDNS Authoritative Server 4.2 will only receive critical security updates and will be end of life after PowerDNS Authoritative Server 4.5 is released.
+PowerDNS Authoritative Server 4.3 will only receive critical security updates and will be end of life after PowerDNS Authoritative Server 4.6 is released.
```
Comment (omoerbeek) [inline docs/appendices/EOL.rst:14]: you might want to leave "security" out here

Comment (Habbie) [reply docs/appendices/EOL.rst:14]: indeed, changed

**Review by omoerbeek — APPROVED** [review]

**Pull request merged**